master/src/ComputerZoomWidget.cpp

```diff
@@ -73,6 +77,71 @@ ComputerZoomWidget::~ComputerZoomWidget()
 
 
 
+bool ComputerZoomWidget::eventFilter( QObject* object, QEvent* event )
+{
+	if( event->type() == QEvent::KeyPress )
+	{
+		vCritical() << "KeyPress Event fired!";
+		Q_EMIT keypressInComputerZoomWidget( );
+
+		const auto screens = m_vncView->computerControlInterface()->screens();
+		const auto key = dynamic_cast<QKeyEvent *>( event )->key();
```

Comment:
You can use `static_cast` since you already checked the event type a few lines before. At least this is what Qt uses everywhere, even though there might be a few places in Veyon's code base which still need to be updated.

Reply:
Okay. Done.